src/test/librbd/test_Groups.cc

```diff
@@ -99,8 +99,9 @@ TEST_F(TestGroup, add_image)
 
   rbd_group_info_t group_info;
   ASSERT_EQ(0, rbd_get_group(image, &group_info, sizeof(group_info)));
-  ASSERT_EQ(0,strcmp("", group_info.name));
+  ASSERT_EQ(0, strcmp("", group_info.name));
   ASSERT_EQ(RBD_GROUP_INVALID_POOL, group_info.pool);
+  free(group_info.name);
```

Comment:
nit: use `rbd_group_info_cleanup` instead?

Reply:
I wonder how I managed to miss that function...